src/shared/constants.ts

```diff
@@ -94,7 +94,9 @@ export const INSIGHTS_TIMESTAMP_FORMAT = 'YYYY-MM-DDTHH:mm:ss.SSSZ'
 /**
  * URI scheme for CloudWatch Logs Virtual Documents
  */
+// TODO: change to a more conventional format like `aws-cloud-watch-logs` or `aws-cwl`
 export const CLOUDWATCH_LOGS_SCHEME = 'awsCloudWatchLogs'
```

Comment:
Changed to `aws-cwl` in the `feature/cwl` branch (for CWL Search). So the TODO can be removed.